benchmarking_hallucination_metrics/benchmark_hallucination_metrics.ipynb

```diff
@@ -0,0 +1,4623 @@
+              "4  31b36417-aad1-412c-b0e5-9c1faaed233f   \n",
+              "\n",
+              "                                             passage  \\\n",
+              "0  Hoping to rebound from the road loss to the Ch...   \n",
+              "1  As of the census of 2000, there were 218,590 p...   \n",
+              "2  Hoping to rebound from the road loss to the Ch...   \n",
+              "3  Hoping to rebound from their tough overtime ro...   \n",
+              "4  As of the census of 2000, there were 218,590 p...   \n",
+              "\n",
+              "                                            question  \\\n",
+              "0  Which team scored the longest field goal kick ...   \n",
+              "1                  How many percent were not  Irish?   \n",
+              "2  How many yards was the second longest field go...   \n",
+              "3                   How long was the last touchdown?   \n",
+              "4  How many in percent from the census weren't Ir...   \n",
+              "\n",
+              "                                              answer label source_ds  \n",
+              "0  ['Rams', 'second', 'Marc Bulger', 'Kevin Curtis']  FAIL      DROP  \n",
+              "1                                               87.1  FAIL      DROP  \n",
+              "2                                                 42  FAIL      DROP  \n",
+              "3                                            18-yard  FAIL      DROP  \n",
+              "4                                               87.1  FAIL      DROP  "
+            ]
+          },
+          "execution_count": 5,
+          "metadata": {},
+          "output_type": "execute_result"
+        }
+      ],
+      "source": [
+        "data.head()"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 6,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 304
+        },
+        "id": "o5g_bsRU3J9T",
+        "outputId": "25188e7f-d29f-4429-e3c4-3f20a4f540e2"
+      },
+      "outputs": [
+        {
+          "data": {
+            "text/plain": [
+              "source_ds\n",
+              "halueval        10000\n",
+              "DROP             1000\n",
+              "pubmedQA         1000\n",
+              "FinanceBench     1000\n",
+              "covidQA          1000\n",
+              "RAGTruth          900\n",
+              "Name: count, dtype: int64"
+            ]
+          },
+          "execution_count": 6,
+          "metadata": {},
+          "output_type": "execute_result"
+        }
+      ],
+      "source": [
+        "data.source_ds.value_counts()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "IuKKUEGQ3J9T"
+      },
+      "source": [
+        "We won't use all of these datasets in our benchmarks due to quality issues in the underlying labels - read the blog post for more details!\n",
```

Comment:
I don't think you actually specify which subset of the datasets we run the benchmark on?

[Here](https://github.com/cleanlab/cleanlab-tools/pull/21/files#diff-6d59619f4398a60b5d3e1dd81273e3c283e22259fd778bb1f935ab9ef85ece72R1412) it seems like we run all of them.

Reply:
Yeah, good point. Dropped